.github/workflows/lint-bpf-checks.yaml

```diff
@@ -133,7 +133,7 @@ jobs:
           fetch-depth: 0
       - name: Run BPF tests with code coverage reporting
         run: |
-          make -C test run_bpf_tests COVER=1 || (echo "Run 'make -C test run_bpf_tests COVER=1' locally to investigate failures"; exit 1)
```

Comment:
Do we need an issue to track this change and remember to revert once the underlying issue is fixed?